### DISCUSSION TITLE
`FileInput` placeholder styles API

### DISCUSSION THREAD

**Discussion by hexcowboy** [mantinedev/mantine#5300] [Q&A]
I'm curious about styling the `FileInput` `placeholder`. It seems to not be a key on `styles` or `classNames`.

I'm also generally curious about adding children to a `FileInput`. My design spec looks like the following and I'm not sure how to accomplish this
![image](https://github.com/mantinedev/mantine/assets/8162609/cbffdba0-b591-440b-8ee4-e71914c59714)

**Comment by rtivital** [comment]
You should probably use Dropzone for this – https://mantine.dev/others/dropzone/
